manager/integration/tests/common.py

```diff
@@ -1950,10 +1952,19 @@ def crash_replica_processes(client, api, volname, replicas=None,
                        "' | grep -v grep | awk '{print $2}'`"
         exec_instance_manager(api, r.instanceManagerName, kill_command)
 
-        # FIXME: this is a work around, should check this in-parallel.
-        #        https://github.com/longhorn/longhorn/issues/4045
+        # XXX: check all replica status in-parallel but it might not make sure
```

Comment:
Out of curiosity, what is `XXX`?

Can we have a couple of tests run for this to see if tests are still flaky?

Reply:
XXX, I used for something needed to be noticed.
Here is one test run this morning [1444](https://ci.longhorn.io/job/private/job/longhorn-tests-regression/1444/), yesterday I run two or three runs for this.
I could run another one to test.

Reply:
You definitely use eclipse :slightly_smiling_face:

Reply:
Ah :slightly_smiling_face:

Reply:
Agree, I would remove `XXX`.

Reply:
updated.